<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
    background-color: transparent;
    color: rgb(219, 242, 39);
	padding-left:3px;
    border: 1px solid #DBF227;
    radius: 15px;
}

QScrollBar:vertical {
    width: 0px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>14</number>
          </property>
          <property name="bottomMargin">
           <number>13</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>66</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 700 12pt &quot;TT Norms Pro&quot;;
	color: rgb(219, 242, 39);
	background-color : transparent;
	border: none
}
</string>
            </property>
            <property name="text">
             <string>Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addToPlaylistButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 700 12 &quot;TT Norms Pro&quot;;
	color:black;
	border:none;
	radius: 20px;
	padding-right:7px;
	padding-left:7px;	height:30px;	
	background-color:#DBF227;
}</string>
            </property>
            <property name="text">
             <string>Add to PlayList</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="create_new_playlist">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 700 12 &quot;TT Norms Pro&quot;;
	color:black;
	border:none;
	radius: 20px;
	padding-right:7px;
	padding-left:7px;
	margin-right:11px;
	height:30px;	
	background-color:#DBF227;
}</string>
            </property>
            <property name="text">
             <string>Create Playlist</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignVCenter">
           <widget class="QPushButton" name="refreshButton">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
	padding-left:5px;
	padding-right:5px;
	padding-top: 5px;
	padding-bottom:5px;
    transition: border 0.15s;
	border-radius:8px;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(7px);
    box-shadow: 5px 3px 10px 5px #dbf227;
}
QPushButton:checked{
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(7px);
    box-shadow: 5px 3px 10px 5px #dbf227;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>13</width>
              <height>13</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 700 12 &quot;TT Norms Pro&quot;;
	color:black;
	border:none;
	radius: 20px;
	padding-right:7px;
	padding-left:7px;	height:20px;	
	background-color:#DBF227;
}</string>
            </property>
            <property name="text">
             <string>Select Music</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="playlist_dropdown_menu">
            <property name="styleSheet">
             <string notr="true">QComboBox {
	radius:20px;
	height:25px;
	margin-left :5px;
	margin-right :5px;
	padding-left:8px;
	background-color:transparent;
	border:1px solid #DBF227;
 	color: #DBF227; 
}
QComboBox::down-arrow {
    color: #DBF227; /* Color of the arrow */
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView {
    background-color: transparent;
    font-size: 13pt;
    font: 700 9pt &quot;TT Norms Pro&quot;;
    border: 1px solid #DBF227;
    border-radius: 15px;
	padding-top:10px;
	padding-left:10px;
}

QListView::item {
    background-color: transparent;
    margin-bottom: 6px;
    color: white;
}
QListWidget::item:selected { 
	color:#042940;
	background-color: yellow; 
}

/* Scrollbar Styles */
QScrollBar:vertical {
    border: none;
    background: transparent;
    width: 10px; /* Width of the scrollbar */
    margin: 16px 0 16px 0;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background-color: #DBF227; /* Color of the scrollbar handle */
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical {
    height: 16px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    height: 16px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}
QScrollBar:horizontal {
    border: none;
    background: transparent;
    height: 10px; /* Height of the scrollbar */
    margin: 0 16px 0 16px;
    border-radius: 4px;
}

QScrollBar::handle:horizontal {
    background-color: #DBF227; /* Color of the scrollbar handle */
    min-width: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal {
    width: 16px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    width: 16px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="AlbumPlaceHolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>250</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    width: 150px;
    height: 150px;
    border-radius: 10px
	background-color: transparent;
	border:none;
    padding: 6px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	font: 700 12pt &quot;TT Norms Pro&quot;;
	color:white;
	background-color : transparent;
	border: none;
	marquee: true
}</string>
          </property>
          <property name="text">
           <string>Music</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="seekbar_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999;
    background: #f0f0f0;
    height: 8px;
    border-radius: 4px;
}

QSlider::sub-page:horizontal {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #9FC131, stop: 1 #DBF227);
    border: 1px solid #777;
    height: 8px;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #fff;
    border: 1px solid #777;
    height: 8px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: #9FC131;
    border: 1px solid #777;
    width: 13px;
    margin-top: -2px;
    margin-bottom: -2px;
    border-radius: 4px;
}

QSlider::handle:horizontal:hover {
    background: #DBF227;
    border: 1px solid #444;
    border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
    background: #bbb;
    border-color: #999;
}

QSlider::add-page:horizontal:disabled {
    background: #eee;
    border-color: #999;
}

QSlider::handle:horizontal:disabled {
    background: #eee;
    border: 1px solid #aaa;
    border-radius: 4px;
}
QSlider{
	background-color:transparent;
	MARGIN-TOP:10px;
	border:none
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="currentSongLabel">
            <property name="styleSheet">
             <string notr="true">QLabel{
	background-color : transparent;
	font: 8pt &quot;TT Norms Pro&quot;;
	color:white;
	border: none
}</string>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="songLengthLabel">
            <property name="styleSheet">
             <string notr="true">QLabel{
	background-color : transparent;
	font: 8pt &quot;TT Norms Pro&quot;;
	color:white;
	border: none
}</string>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QPushButton" name="shuffle_button">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: transparent;
    margin-left: 10px;
    border: none;
	padding-left:2px;
	padding-right:2px;
	padding-top: 2px;
	padding-bottom:2px;
    transition: border 0.15s;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
QPushButton:checked{
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:transparent;
	margin-left:13px;
	border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:transparent;
	margin-right:5px;
	margin-left:5px;
	border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forwButton">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:transparent;
	margin-right:13px;
	border: none
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="repeatButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
	padding-left:2px;
	padding-right:2px;
	padding-top: 3px;
	margin-right:10px;
	padding-bottom:3px;
    transition: border 0.15s;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
QPushButton:checked{
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>180</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="volume_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
	padding-top: 2px;
	padding-bottom:2px;
    transition: border 0.15s;
	margin-right:15px;
	margin-left:5px;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
QPushButton:checked{
    background-color: rgba(255, 255, 255, 0.1);
    backdrop-filter: blur(10px);
    box-shadow: 0 0 10px 5px #dbf227;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item alignment="Qt::AlignBottom">
           <widget class="QSlider" name="volumesilder">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999;
    background: #f0f0f0;
    height: 8px;
    border-radius: 4px;
}

QSlider::sub-page:horizontal {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #9FC131, stop: 1 #DBF227);
    border: 1px solid #777;
    height: 8px;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #fff;
    border: 1px solid #777;
    height: 8px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: #9FC131;
    border: 1px solid #777;
    width: 13px;
    margin-top: -2px;
    margin-bottom: -2px;
    border-radius: 4px;
}

QSlider::handle:horizontal:hover {
    background: #DBF227;
    border: 1px solid #444;
    border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
    background: #bbb;
    border-color: #999;
}

QSlider::add-page:horizontal:disabled {
    background: #eee;
    border-color: #999;
}

QSlider::handle:horizontal:disabled {
    background: #eee;
    border: 1px solid #aaa;
    border-radius: 4px;
}
QSlider{
	background-color:transparent;
	border:none
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
